### ISSUE TITLE
Setup config

### ISSUE THREAD

**Issue by fugokidi** [chazmcgarvey/vimcoder#13]
I want to compile and test locally, the Makefile is empty by default.
How to make Makefile work?
At startup, vim is opening up both c++ and java template.
How to make vimcoder give me only C++ template?
Thank you.

**Comment by chazmcgarvey** [comment]
It seems like the topcoder applet may be doing something odd with regard to the plug-in.

Can you provide more information about your setup?
- operating system and version
- TopCoder applet version (appears on log-in screen)
- VimCoder version (part of jar filename)
- java -version
- vim --version

**Comment by fugokidi** [comment]
Thank you for the reply. I thought you were busy and could not reply me. I've just seen the reply now.

Second problem with me is solved.
I didn't set the default language before, that's why it gives me Java template. It's fixed.

But Makefile is still empty.
I cannot compile and run offline.

* OS version => OSX 10.10.2 (Yosemite)
* TopCoder Applet version => 7.1.3
* VimCoder version => 0.3.6 (I built it with make, make jar)
* java --version => Version 8 update 40
* vim --version => 7.4

**Comment by fugokidi** [comment]
Dear Sir,
The problem with me is fixed. Actually the Makefile was empty because the Java was first priority when the problem is open, when I chose C++, I've two templates opened and an empty Makefile. Now, I've set the default language and everything is good, your plugin is awesome.

However, *it would be nice, anytime C++ is chosen, the correct Makefile is created (Not the empty one) regardless of Java has been chosen or not*. I don't know how to fix it.
